frontend/src/components/LandingPage/LandingPage.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react'
 //Import components that you create here
+import './LandingPage.css';
+import TeamPage from './TeamPage/TeamPage';
 import Explanation from './Explanation/Explanation'
+import StyleSheet from '../StyleSheet/StyleSheet'
 
 function LandingPage() {
     return (
         //Use components that you import here
         <>
-            <Explanation />
+            <StyleSheet />
```

Comment:
When we merge this well have to make sure to change this back.

Reply:
Yes, sorry about that! I can take that out now